doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,297 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/overview.png
-    :align: center
-    :width: 20%
+See :ref:`Ray Metrics <ray-metrics>` to learn how to set up Prometheus and Grafana.
 
-.. _dash-node-view:
+If you need to customize the port on which the dashboard will run, you can pass
+the ``--dashboard-port`` argument with ``ray start`` in the command line, or you can pass the
+keyword argument ``dashboard_port`` in your call to ``ray.init()``.
 
-Node View
-~~~~~~~~~
+How to Guides
+-------------
 
-The Node view lets you see resource utilization information on a per-node and per-worker basis.
-This also shows the assignment of GPU resources to specific actors or tasks.
+.. _dash-workflow-critial-system-metrics:
 
-In addition, the machine view lets you see **logs** for a node or a worker.
+View system metrics
+~~~~~~~~~~~~~~~~~~~
+
+Ray exports default metrics which are available from the :ref:`Metrics View <dash-metrics-view>`. Here are some available example metrics.
+
+- The tasks, actors, and placement groups broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
+
+.. _dash-workflow-resource-utilization:
+
+Comparing the hardware and logical resource utilization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ray requires users to specify the number of :ref:`resources <logical-resources>` their tasks and actors will use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``. 
+These values are used for scheduling, but may not always match the actual resource utilization. 
+
+If the logical resources don't match the physical resource usage, it can lead to the under-utilization of the cluster because the number of tasks and actors are throttled by
+the logical resource requirements.
+
+You can check the :ref:`Metrics View <dash-metrics-view>` to compare logical and hardware resource utilization. 
+
+Let's see an example.
+
+.. code-block:: python
+
+    @ray.remote(num_cpus=1)
+    def task():
+        import time
+        time.sleep(30)
+    
+    # Launch 30 tasks
+    ray.get([task.remote() for _ in range(30)])
+
+When you run the below code that executes many tasks that sleep, 
+you will see from the metrics page that there are many logical CPU allocations but little to no hardware CPU usage.
+It is because the logical resource requirement is 1 CPU for each task, but each task uses nearly no CPU because it just sleeps.
+
+TODO-SANG Add images
+
+resource specification for those tasks will lead to wasted physical resources. To address this issue, let's adjust the ``num_cpus`` value to 0.1 so that other tasks or actors can be scheduled to leverage the hardware resources.
```

Comment:
```suggestion
Resource specification for those tasks will lead to wasted physical resources. To address this issue, let's adjust the ``num_cpus`` value to 0.1 so that other tasks or actors can be scheduled to leverage the hardware resources.
```